packaging/bundle-validation/validate.sh

```diff
@@ -376,7 +376,7 @@ if [ "$?" -ne 0 ]; then
 fi
 echo "::warning::validate.sh done validating utilities slim bundle"
 
-if [[ ${JAVA_RUNTIME_VERSION} == 'openjdk8' && ${SCALA_PROFILE} != 'scala-2.13' && ! "${FLINK_HOME}" == *"1.18"* ]]; then
+if [[ ${JAVA_RUNTIME_VERSION} == 'openjdk8' && ${SCALA_PROFILE} != 'scala-2.13' ]]; then
```

Comment:
While the change looks good, the Flink bundle validation is still not executed on Flink 1.18, as the corresponding bundle validation profile is coupled with `scala-2.13` (and I don’t see Flink bundle validation success message in GH CI). Could you try adding or changing a profile to let Flink 1.18 bundle validation run and see if it succeeds?
```
- scalaProfile: 'scala-2.13'
            flinkProfile: 'flink1.18'
            flinkAvroVersion: '1.11.1'
            sparkProfile: 'spark3.5'
            sparkRuntime: 'spark3.5.1'
```

Reply:
![image](https://github.com/user-attachments/assets/5f67ae99-2468-4228-89a9-57c3c0d17ead)

Reply:
Sg